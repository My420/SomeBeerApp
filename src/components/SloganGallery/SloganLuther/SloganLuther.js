import React from 'react';
import styles from './SloganLuther.module.scss';
import Picture from '../../Picture/Picture';
import imagePC from './img/beer_glass@PC.jpg';
import imageTablet from './img/beer_glass@Tablet.jpg';
import imageMobile from './img/beer_glass@Mobile.jpg';

const SloganLuther = () => {
  return (
    <article className={styles.container}>
      <h5 className="visually-hidden"> Martin Luther quote</h5>
      <Picture
        className={styles.image}
        alt="beer glass"
        pcSrc={imagePC}
        tabletSrc={imageTablet}
        mobileSrc={imageMobile}
      />
      <blockquote className={styles.quote}>
        <p className={styles.text}>
          «Whoever drinks beer, he is quick to sleep; whoever sleeps long, does
          not sin; whoever does not sin, enters Heaven! Thus, let us drink
          beer!»
        </p>
        <cite className={styles.author}>~ Martin Luther</cite>
      </blockquote>
    </article>
  );
};

export default SloganLuther;
