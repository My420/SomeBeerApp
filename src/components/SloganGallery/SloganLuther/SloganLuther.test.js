import React from 'react';
import { shallow } from 'enzyme';
import SloganLuther from './SloganLuther';

describe('Test <SloganLuther/> component', () => {
  const wrapper = shallow(<SloganLuther />);

  test('should correct render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
