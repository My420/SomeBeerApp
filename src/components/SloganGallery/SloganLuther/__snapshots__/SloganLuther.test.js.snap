// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <SloganLuther/> component should correct render component 1`] = `
<article
  className="container"
>
  <h5
    className="visually-hidden"
  >
     Martin Luther quote
  </h5>
  <Picture
    alt="beer glass"
    className="image"
    mobileSrc="beer_glass@Mobile.jpg"
    pcSrc="beer_glass@PC.jpg"
    tabletSrc="beer_glass@Tablet.jpg"
  />
  <blockquote
    className="quote"
  >
    <p
      className="text"
    >
      «Whoever drinks beer, he is quick to sleep; whoever sleeps long, does not sin; whoever does not sin, enters Heaven! Thus, let us drink beer!»
    </p>
    <cite
      className="author"
    >
      ~ Martin Luther
    </cite>
  </blockquote>
</article>
`;
